static/js/lib/search.js

```diff
@@ -620,8 +620,8 @@ export const buildLearnQuery = (
     if (!emptyOrNil(text)) {
       // $FlowFixMe: if we get this far, text is not null
       builder = builder.rawOption("suggest", buildSuggestQuery(text))
-    } else {
-      builder.rawOption("sort", buildDefaultSort())
+    } else if (facetClauses.length === 0 && types === LR_TYPE_ALL) {
```

Comment:
Strictly equals in JS can fail for lists even if the elements match if the two references are different objects, `R.equals` is safer: https://ramdajs.com/docs/#equals